Source/Laser/LaserProfilesImpl/LaserProfileFromFile.cpp

```diff
@@ -578,3 +522,178 @@ WarpXLaserProfiles::FromFileLaserProfile::internal_fill_amplitude_uniform_binary
         }
     );
 }
+
+void
```

Comment:
Could you move this function back to its original position?
It would make it easier to see the differences on Github.

Reply:
Also: would it make sense to split this function into two separate functions:
```
internal_fill_amplitude_uniform_cartesian
```
```
internal_fill_amplitude_uniform_cylindrical
```